cli/step_environment.go

```diff
@@ -16,67 +17,72 @@ type prepareStepInputParams struct {
 	isCIMode, isPullRequestMode bool
 }
 
-func prepareStepEnvironment(params prepareStepInputParams, envSource env.EnvironmentSource) ([]envmanModels.EnvironmentItemModel, map[string]string, error) {
+func prepareStepEnvironment(params prepareStepInputParams, envSource env.EnvironmentSource) ([]envmanModels.EnvironmentItemModel, map[string]string, map[string]interface{}, error) {
```

Comment:
nit: I find it hard to understand the 3 return values of this function since the 2nd and 3rd are both `map[string]string` types. Named return types or a doc comment above the function would help here.